### PR TITLE
Ibrs

### PR DESCRIPTION
**Pull request by mykaul**
Add several CPUs with IBRS capability.

### CODE REVIEW

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
I'll verify it on a slave that has *IBRS CPU and then merge.

File: ovirtlago/data/ovirt_cpu_map.yaml [hunk 1]
```diff
@@ -8,12 +8,20 @@ Intel:
...
+  Westmere-IBRS: Intel Westmere-IBRS Family
   SandyBridge: Intel SandyBridge Family
+  SandyBridge-IBRS: Intel SandyBridg-IBRS Family
   IvyBridge: Intel SandyBridge Family
   Haswell-noTSX: Intel Haswell-noTSX Family
+  Haswell-IBRS: Intel Haswell-IBRS Family
+  Haswell-noTSX-IBRS: Intel Haswell-noTSX-IBRS Family
   Haswell: Intel Haswell Family
   Broadwell-noTSX: Intel Broadwell-noTSX Family
+  Broadwell-IBRS: Intel Broadwell-IBRS Faily
```
Comment (gbenhaim) [inline ovirtlago/data/ovirt_cpu_map.yaml]: s/Faily/Family

Comment (mykaul) [reply ovirtlago/data/ovirt_cpu_map.yaml]: Done.

**Comment by eedri** [comment]
We have to merge this ASAP since all lago slaves were upgraded to newest kernel and its failing tests now.
@gbenhaim you mentioned an open Libvirt bug that might be an issue if we release this fix, can you add more info on it and if its still a risk?

**Comment by gbenhaim** [comment]
I saw that *-IBRS CPU doesn't propagate to the VM. In the VM I see that the CPU model is *.

For example, if the host's CPU is `Haswell-noTSX-IBRS` the VM's CPU is `Haswell-noTSX`.

We decide on the CPU family for the cluster based on the host's CPU or the configuration in LagoInitFile (we assume the usage of `host-passthrough`), which cause add_cluster to fail because the cpu_family of the cluster is *-IBRS while the host VM CPU is *.

The description above affects only  4.1. In 4.2 the CPU is auto-detected when the first host is added to the cluster.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1533125

**Comment by gbenhaim** [comment]
merged in https://github.com/lago-project/lago-ost-plugin/pull/32

**Pull request closed without merging**